remotespark/kernels/kernelmagics.py

```diff
@@ -64,6 +66,7 @@ def __init__(self, shell, data=None, spark_events=None):
     @handle_expected_exceptions
     @_event
     def help(self, line, cell="", local_ns=None):
+        self._assure_cell_body_is_empty(KernelMagics.help.__name__, cell)
```

Comment:
There are some magics that don't receive parameters (e.g. spark). If someone wrote:

```
%% spark a = 1
print a
```

What error will they get, and do they have any hope of fixing it after this error message?